### ISSUE TITLE
savon.expects(:action) not working

### ISSUE THREAD

**Issue by betelgeuse** [savonrb/savon_spec#12]
When I don't specify a return a value savon_spec fails calling code  for nil.

```
#<NoMethodError: undefined method `code' for nil:NilClass>
```

```
"/Users/betelgeuse/.rvm/gems/ruby-1.9.3-p125/gems/savon-0.9.7/lib/savon/soap/request.rb:52:in `with_logging'",
 "/Users/betelgeuse/.rvm/gems/ruby-1.9.3-p125/gems/savon-0.9.7/lib/savon/soap/request.rb:31:in `response'",
 "/Users/betelgeuse/.rvm/gems/ruby-1.9.3-p125/gems/savon-0.9.7/lib/savon/client.rb:79:in `request'",
```

**Comment by rubiii** [comment]
that's because you need to return something.
https://github.com/rubiii/savon_spec/blob/master/lib/savon/spec/mock.rb#L45

**Comment by betelgeuse** [comment]
@rubiii then the README is wrong:
https://github.com/rubiii/savon_spec/commit/4303b0e71fd0c265bd3c5a0a4ddcf058d8ad4260#L0R39

**Comment by rubiii** [comment]
right. thanks for spotting this. let me see if it's possible to leave out the call to `returns`.

**Comment by jedbeard** [comment]
EDIT: moved to #15

**Comment by rubiii** [comment]
mr. @jedbeard is there a reason why you added your problem to this issue?
it sounds like a whole different issue to me.

**Comment by jedbeard** [comment]
I've moved the issue, but my problem is still alive!!

**Comment by rubiii** [comment]
i'll look into it :)

**Comment by rubiii** [comment]
this will be fixed with savon 2.0. please [read the documentation](http://savonrb.com/version2.html) and give it a try if you can.
